core/lib/zksync_core/src/eth_sender/eth_tx_aggregator.rs

```diff
@@ -39,8 +39,9 @@ pub struct MulticallData {
 /// Such as CommitBlocks, PublishProofBlocksOnchain and ExecuteBlock
 /// These eth_txs will be used as a queue for generating signed txs and send them later
 #[derive(Debug)]
-pub struct EthTxAggregator {
+pub struct EthTxAggregator<E> {
```

Comment:
Cannot get rid of `E` right now, since `EthInterface` is not object-safe. It'll be fixed separately though, and later I'll get back here.